apps/studio/src/features/editing-experience/components/MoveResourceModal/MoveResourceModal.tsx

```diff
@@ -188,24 +188,26 @@ const MoveResourceContent = withSuspense(
                 </Flex>
               )}
               {data?.pages.map(({ items }) =>
-                items.map((child) => {
-                  return (
-                    <MoveItem
-                      {...child}
-                      key={child.id}
-                      onChangeResourceId={() => {
-                        setResourceStack((prev) => [
-                          ...prev,
-                          {
-                            ...child,
-                            parentId: curResourceId ?? null,
-                            resourceId: child.id,
-                          },
-                        ])
-                      }}
-                    />
-                  )
-                }),
+                items
+                  .filter((child) => child.id !== movedItem?.resourceId)
```

Comment:
feel like shouldn't filter it out. can we set it to disabled state for (like for pages) if the resource is moving to itself?

Reply:
sounds good, lemme make this change!